src/Profiles/Fec/DataPages/ANTPLUS_FecBasicResistanceDataPage.cpp

```diff
@@ -0,0 +1,28 @@
+#include <Profiles/Fec/DataPages/ANTPLUS_FecBasicResistanceDataPage.h>
+#include <Profiles/Fec/ANTPLUS_FecPrivateDefines.h>
+
+#define POWER_BYTE 7
+
+template<class T>
+FecBaseBasicResistanceDataPage<T>::FecBaseBasicResistanceDataPage() :
+    CoreDataPage<T>() {}
+
+template<class T>
+uint8_t FecBaseBasicResistanceDataPage<T>::getPower() {

```

Comment:
where does "power" come from? This field is labeled as "total resistance" in my doc